src/tests/ftest/launch.py

```diff
@@ -2336,6 +2342,54 @@ def _reset_server_storage(test):
             logger.debug("  Skipping resetting server storage - no server hosts")
         return 0
 
+    @staticmethod
+    def _cleanup_procs(test, do_stop):
+        """Cleanup any processes left running on remote nodes.
+
+        Args:
+            test (TestInfo): the test information
+            do_stop (bool): whether to stop the processes
+
+        Returns:
+            int: status code: 0 = success; 4096 if processes were found
+
+        """
+        any_found = False
+
+        hosts = test.host_info.all_hosts
+        proc_pattern = "|".join(PROCS_TO_CLEANUP)
+        logger.info("Looking for running processes: %s", proc_pattern)
+        pgrep_cmd = f"pgrep --list-full '{proc_pattern}'"
+        pgrep_result = run_remote(logger, hosts, pgrep_cmd)
+        if pgrep_result.passed_hosts:
+            if do_stop:
+                logger.info("Killing running processes: %s", proc_pattern)
+                pkill_cmd = f"sudo -n pkill -e --signal KILL '{proc_pattern}'"
+                pkill_result = run_remote(logger, pgrep_result.passed_hosts, pkill_cmd)
+                if not pkill_result.passed:
+                    # For now, just warn and continue
+                    logger.warning(
+                        "Failed to kill running processes on: %s", pkill_result.failed_hosts)
```

Comment:
In order to get a clear entry in the results.html file, I'd suggest adding a `Launch._warn_test()` method - very similar to  `Launch._fail_test()` to call in place of logger.warning().

Reply:
Agree. Thanks. Done

Reply:
Done. See the latest updates. Notice in the latest run that we got some WARN first, but ERROR is reported: https://build.hpdd.intel.com/blue/organizations/jenkins/daos-stack%2Fdaos/detail/PR-11341/12/tests